vitrina/datasets/forms.py

```diff
@@ -402,8 +404,19 @@ def __init__(self, request=None, organization=None, *args, **kwargs):
             Field("creator"),
             Field("publisher"),
             Field("information_system_type"),
+            Field("information_system_publisher"),
+            Field("information_system_creator")
         )
 
+        self.fields['information_system_publisher'].required = True
+        self.fields['information_system_publisher'].help_text = _(
+            "(dct:publisher) Ši savybė nurodo subjektą (organizaciją), atsakingą už IS prieinamumą"
+            )
+        self.fields['information_system_creator'].required = True
+        self.fields['information_system_creator'].help_text = _(
+            "(dct:creator) Subjektas, atsakingas už IS parengimą."
+            )
```

Comment:
Help texts should follow the convention we agreed upon yesterday:

`<description>. Corresponds/Adheres to <dcat reference>`

So in your case:

```suggestion
        self.fields['information_system_publisher'].help_text = _(
            "Ši savybė nurodo subjektą (organizaciją), atsakingą už IS prieinamumą. Atitinka dct:publisher"
            )
        self.fields['information_system_creator'].required = True
        self.fields['information_system_creator'].help_text = _(
            "Subjektas, atsakingas už IS parengimą. Atitinka dct:creator"
            )
```

Refer to this:
- https://github.com/atviriduomenys/katalogas/pull/1805/files